### PR TITLE
Customer order

### PR DESCRIPTION
**Pull request by AhmedAbdallah125**
Screen Shot 2022-06-05 at 9 47 04 PM
Screen Shot 2022-06-05 at 9 46 41 PM
Screen Shot 2022-06-05 at 9 46 05 PM

### CODE REVIEW

File: app/src/main/res/navigation/mobile_navigation.xml [hunk 1]
```diff
@@ -111,4 +111,9 @@
             android:id="@+id/action_navigation_products_to_productsDetailsFragment"
             app:destination="@id/productsDetailsFragment" />
     </fragment>
+    <fragment
+        android:id="@+id/ordersFragment"
+        android:name="com.iti.android.team1.ebuy.ui.orders.view.OrdersFragment"
+        android:label="fragment_orders"
```
Comment (AbdulrahmanAli19) [inline app/src/main/res/navigation/mobile_navigation.xml:117]: good work Ahmed 💪, you just forgat to add a label in the strings.xml file

**Review by AbdulrahmanAli19 — CHANGES_REQUESTED** [review]
good work Ahmed 💪, you just forgat to add a label in the strings.xml file

File: app/src/main/res/navigation/mobile_navigation.xml (hunk 1 above)
Comment (AhmedAbdallah125) [reply app/src/main/res/navigation/mobile_navigation.xml:117]: ok

**Review by AbdulrahmanAli19 — APPROVED** [review]

File: app/src/main/res/navigation/mobile_navigation.xml (hunk 1 above)
Comment (AhmedAbdallah125) [reply app/src/main/res/navigation/mobile_navigation.xml]: Done

**Review by Mina-Wael — APPROVED** [review]

**Pull request merged**